birdhouse/README.rst

```diff
@@ -16,6 +17,38 @@ Requirements:
 
 * Install latest docker-ce and docker-compose for the chosen distro (not the
   version from the distro).
+  
+* Have a real SSL Certificate, self-signed SSL Certificate do not work properly.
+  Let's Encrypt offers free SSL Certificate.
+
+* If using Let's Encrypt, port 80 and 443 and hostname should be accessible publicly over the internet.
```

Comment:
Can we add the following just to make sure that it's clear this is required *in order* to register a SSL certificate with Let's Encrypt:

> If using Let's Encrypt, port 80 and 443 and hostname should be accessible publicly over the internet before requesting a certificate with Let's Encrypt. Let's Encrypt will need to access your hostname at port 80 and 443 in order to verify and provide the SSL certificate.

Reply:
fixed 186c3ecb2e14c74fabe3c5cb4ff6344087bcf555, thanks for the much improved wording.  I tend to be too much "short and concise" sometime.